src/server/system_services/account_server.js

```diff
@@ -6,7 +6,8 @@ const _ = require('lodash');
 const net = require('net');
 const chance = require('chance')();
 const GoogleStorage = require('../../util/google_storage_wrap');
-const bcrypt = require('bcrypt');
+const node_hash = require('../../util/account_password_hash');
```

Comment:
I would call this new module `password_utils`
```suggestion
const password_utils = require('../../util/password_utils');
```